src/audio/__tests__/AudioProvider.test.tsx

```diff
@@ -0,0 +1,47 @@
+import { fireEvent, render } from '@testing-library/react';
+import { RecoilRoot } from 'recoil';
+import useAudio from '../__mocks__/use-audio.ts';
+import { AudioProvider } from '../AudioProvider';
+
+vi.mock('../__mocks__/useAudio.ts');
```

Comment:
You should be mocking the actual useAudio rather than the mock file

Reply:
test is updated to match this requirement